tests/compatibility-test.py

```diff
@@ -86,8 +87,9 @@ class RayFTTestCase(unittest.TestCase):
     cluster_template = CONST.REPO_ROOT.joinpath("tests/config/ray-cluster.ray-ft.yaml.template")
     ray_cluster_ns = "default"
 
-    @classmethod
```

Comment:
How about creating a new test for cleaning up Redis? That is, `test_ray_serve` -> `test_detached_actor` -> `test_cleanup_redis`.

Reply:
That’s was my first attempt. Unfortunately, the `test_cleanup_redis` was going to be the first test case that run by the Python `unittest` framework and the test case was going to delete the RayCluster CR required by the rest of the cases.

The execution order of tests under a `unittest.TestCase` is alphabetical by their method names. Therefore, if we want to use a standalone case to test redis cleanup, we may need to encode sequence into all test names. For example: `test_01_ray_serve` -> `test_02_detached_actor` -> `test_ 99_cleanup_redis`